decidim-admin/app/commands/decidim/admin/create_import.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Admin
+    class CreateImport < Rectify::Command
+      def initialize(form)
+        @form = form
+      end
+
+      def call
+        return broadcast(:invalid) if form.invalid?
+
+        form.context[:user_group] = user_group
+
+        importer = importer_for(form.file_path, form.mime_type)
+        imported_data = importer.prepare
+
+        return broadcast(:invalid_lines, importer.invalid_lines) unless importer.invalid_lines.empty?
+
```

Comment:
Maybe consider moving this logic also to the form and adding a validation there?

Reply:
Yep. Form validates invalid_lines now.